### ISSUE TITLE
Wrap HOMER for IUC?

### ISSUE THREAD

**Issue by mblue9** [galaxyproject/tools-iuc#1892]
>[HOMER]( http://homer.ucsd.edu/homer/)(Hypergeometric Optimization of Motif EnRichment) is a suite of tools for Motif Discovery and next-gen sequencing analysis.  It is a collection of command line programs for unix-style operating systems written in Perl and C++. HOMER was primarily written as a de novo motif discovery algorithm and is well suited for finding 8-20 bp motifs in large scale genomics data.  HOMER contains many useful tools for analyzing ChIP-Seq, GRO-Seq, RNA-Seq, DNase-Seq, Hi-C and numerous other types of functional genomics sequencing data sets.

As per conversation on Gitter, there are a few old wrappers already, could/should they be updated? 

I think the annotatePeaks module especially could be useful, as afaik, there's no other tool for easily adding gene names to peaks e.g. from macs2 or diffbind (please correct me if I'm wrong as I need a tool for that atm)

Current HOMER wrappers:
https://github.com/bgruening/galaxytools/tree/master/tools/homer
https://toolshed.g2.bx.psu.edu/view/kevyin/homer/f0b5827b6051 (this one has the annotatePeaks module)

ping @jennaj, @bgruening, @davebx

**Comment by mblue9** [comment]
Thanks for the info @mvdbeek! That's useful to know. I've ended up wrapping ChIPseeker (https://github.com/bgruening/galaxytools/tree/master/tools/chipseeker) and that seems to work well for annotating peaks. I thought about using bedtools intersect or closest, but then one issue for us is, what annotation file to provide for users to intersect with (and how to version/maintain), and I'd rather not have users faff around with extra manipulation steps/regexs for this if they don't have to. I was looking for a one-step annotation solution and with chipseeker I think I found it :)
